packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -38,12 +44,36 @@ class SessionReplay implements SessionReplayEnrichmentPlugin {
   maxPersistedEventsSize = MAX_EVENT_LIST_SIZE_IN_BYTES;
   interval = MIN_INTERVAL;
   timeAtLastSend: number | null = null;
+  options: SessionReplayOptions;
+  shouldRecord = true;
+
+  constructor(options?: SessionReplayOptions) {
+    this.options = { ...options };
+  }
 
   async setup(config: BrowserConfig) {
     config.loggerProvider.log('Installing @amplitude/plugin-session-replay.');
 
     this.config = config;
+    this.config.sessionId = config.sessionId;
     this.storageKey = `${STORAGE_PREFIX}_${this.config.apiKey.substring(0, 10)}`;
+
+    if (typeof config.defaultTracking === 'boolean') {
+      if (config.defaultTracking === false) {
+        config.defaultTracking = {
+          pageViews: false,
+          formInteractions: false,
+          fileDownloads: false,
+          sessions: true,
+        };
+      }
+    } else {
+      config.defaultTracking = {
+        ...config.defaultTracking,
+        sessions: true,
```

Comment:
This forces sessions to be tracked when using session replay plugin. I think it's the right behavior, but i wonder if we need to document this. Why do customers want this off?

Reply:
Yeah this was a suggestion from Kevin, as the plugin stands now I do think we need to require this as a part of using the plugin. I'll add a note to the README

Reply:
I'm not sure why people would want this off, in browser SDK 2.0 they changed it to be on by default, so I wonder if it's an edge case for customers